client/src/components/AboutTheProduct/AboutTheProductImg/Analysis_White.svg

```diff
@@ -0,0 +1,56 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="174" height="174" viewBox="0 0 174 174">
+    <defs>
```

Comment:
I am not completely sure what this file is but if this is image, it should be placed in client/pubilc/img